module/merintr/mermain.c

```diff
@@ -134,13 +166,17 @@ void InterfaceRedrawModule(HDC hdc)
   StatsDrawBorder();
   StatsMainRedraw();
   StatsDraw();
+
   if( StatsGetCurrentGroup() == STATS_INVENTORY )
   {
     InvalidateRect( GetHwndInv(), NULL, FALSE );
     ShowInventory(TRUE);
     InventoryRedraw();
   }
+  
+  RestoreActiveGroup();
```

Comment:
Is the order of this correct with the lines immediately preceding?  If the current group is INVENTORY, we show the inventory, then we immediately restore the group to some other value?